### PR TITLE
Adds internal/articles input to update author

### PR DESCRIPTION
**Pull request by jessleenyc**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Adds an input to internal/articles to update author id. I think it's worth moving some of these items to its own /edit page eventually the way we do for more destructive changes w/ internal/users.

### CODE REVIEW

File: app/views/internal/articles/_individual_article.html.erb [hunk 1]
```diff
@@ -44,14 +44,16 @@
...
-		<a href="/internal/users/<%= article.user_id %>">@<%= article.user&.username %></a> ❤️ <%= article.positive_reactions_count %> | 💬 <%= article.comments_count %> | ID: <a href="/internal/articles/<%= article.id %>"><%= article.id %></a>
+		<a href="/internal/users/<%= article.user_id %>">@<%= article.user&.username %></a> ❤️ <%= article.positive_reactions_count %> | 💬 <%= article.comments_count %>
 		<br/><br/>
 		<div class="inner-row">
 			<form action="/internal/articles/<%= article.id %>" accept-charset="UTF-8" method="post">
 				<input name="utf8" type="hidden" value="✓">
 				<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>" />
 				<input type="hidden" name="_method" value="patch" />
 				Featured Number: <input name="article[featured_number]" value="<%= article.featured_number %>" id="featured_number_<%= article.id %>" style="max-width:140px"><button onclick="timeNow(<%= article.id %>);return false;">☝️</button><button onclick="sink(<%= article.id %>);return false;">👇 SPAM!</button>
+				<a href="/internal/articles/<%= article.id %>/edit">
```
Comment (benhalpern) [inline app/views/internal/articles/_individual_article.html.erb]: Missing closing `</a>` here causing everything thereafter to be part of the link



Was this link even supposed to be there?

File: app/models/article.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ class Article < ApplicationRecord
 
   acts_as_taggable_on :tags
 
-  attr_accessor :publish_under_org
+  attr_accessor :publish_under_org, :new_author
```
Comment (benhalpern) [inline app/models/article.rb]: Is `new_author` necessary vs using `user_id`?

File: app/views/internal/articles/_individual_article.html.erb (hunk 1 above)
Comment (jessleenyc) [reply app/views/internal/articles/_individual_article.html.erb]: haha whoops! i had caught that but forgot to push again.

File: app/models/article.rb (hunk 2 above)
Comment (jessleenyc) [reply app/models/article.rb]: yep good call

**Pull request merged**